library/ssh_copy_id.py

```diff
@@ -94,7 +98,8 @@ def run_module():
     username = module.params['username']
     password = module.params['password']
     public_key = module.params['ssh_public_key']
-
```

Comment:
Could this carriage return be removed? This way then all the local variables are combined together.

Reply:
It's done.